webapp/src/Twig/TwigExtension.php

```diff
@@ -68,6 +70,7 @@ public function __construct(
         $this->tokenStorage         = $tokenStorage;
         $this->authorizationChecker = $authorizationChecker;
         $this->projectDir           = $projectDir;
+        $this->debug                = $debug;
```

Comment:
Can just use `app.debug` instead in the template instead of adding a new global?